frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelSecondaryTabs.tsx

```diff
@@ -14,18 +12,23 @@ import { BreakdownFilter } from 'scenes/insights/BreakdownFilter'
 import { CloseButton } from 'lib/components/CloseButton'
 import { BreakdownType, FunnelVizType } from '~/types'
 import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import { FunnelConversionWindowFilter } from 'scenes/insights/InsightTabs/FunnelTab/FunnelConversionWindowFilter'
 
 export function FunnelSecondaryTabs(): JSX.Element | null {
-    const { activeView } = useValues(insightLogic)
     const { filters, clickhouseFeaturesEnabled } = useValues(funnelLogic)
     const { setFilters } = useActions(funnelLogic)
-    if (activeView !== ViewType.FUNNELS) {
-        return null
-    }
+    const showConversionWindowFilter =
+        filters.funnel_viz_type === FunnelVizType.TimeToConvert || filters.funnel_viz_type === FunnelVizType.Trends
```

Comment:
Hey Paolo could you clarify what you mean here? I believe we already have the option to filter steps in `FunnelHistogram here` because it's local to only the time conversion viz - https://github.com/PostHog/posthog/blob/d93c2ab837d2af471bbd9a3a6cf2b9585f1c47ce/frontend/src/scenes/funnels/FunnelHistogram.tsx#L15-L15

Reply:
I believe Paolo is interested in why conversion window config is hidden in the "Conversion steps" viz and only shown in "Time to convert" & "Historical trends". I'm pretty curious too, as I think this _is_ supported in the backend for steps just the same, so I propose removing `showConversionWindowFilter` as a condition.